File: Chapter 5/TestMath.vb
Public Class Math

    Function swap(ByVal a As Integer, ByVal b As Integer)

        Dim temp As Integer

        temp = a
        a = b
        b = temp

        Return temp

    End Function

End Class

Module Module1

    Sub Main()

        Dim myMath As New Math()
        myMath.swap(2, 3)

        System.Console.ReadLine()

    End Sub

End Module

File: Chapter 5/TestMath.java
public class TestMath {
    
	public static void main(String[] args) {
    		
		Math myMath = new Math();
		myMath.swap(2, 3);
    		System.out.println("Hello World!");
	}
}

public class Math {

	public int swap(int a, int b){

		int temp = 0;

		temp = a;
		a = b;
		b = temp;

		return temp;

	}

}